docs/Configuration/Backups.md

```diff
@@ -129,3 +129,18 @@ Native restore from backup
 2. Install PaperTrail
 3. Check new installation and enter existing database details
 
+## Transfer an instance of PaperTrail to another host
```

Comment:
I think include the work migrate in here somewhere for searchability.

I would avoid duplicating details.
 e.g Backup and then restore the DB to the new server (See xxxx)
Install papertrail on the new server (see xxx)